fastify.js

```diff
@@ -127,6 +127,8 @@ function build (options) {
   fastify.addHook = addHook
   fastify._hooks = new Hooks()
 
+  fastify._onRouteHooks = []
```

Comment:
can you avoid attaching this to the instance itself?

Reply:
But how we can test this?

Reply:
You test that the hook function gets called, not that you add things to it.